### PR TITLE
Navigation: Add font size attribute

### PR DESCRIPTION
**Pull request by Copons**
## Description

Adds the font size style attributes (both named and custom) to the Navigation block.

## How has this been tested?
Tested on macOS 10.15.2, Chrome 78, Gutenberg Docker environment, Twenty Twenty theme.

## Screenshots

| Before | After |
| - | - |
| Screenshot 2019-12-13 at 15 53 36 | Screenshot 2019-12-13 at 15 54 02 |
| Screenshot 2019-12-13 at 15 51 58 | Screenshot 2019-12-13 at 15 54 22 |

## Testing instructions

- Edit a post and add a Navigation block.
- Try changing the block's font size.
- Make sure the block looks as expected.
- Preview the post, and make sure the block looks as expected in the front end too.
- Repeat the previous steps, but this time check if the font size _custom_ attribute work as expected.

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR. .

### CODE REVIEW

**Comment by Copons** [comment]
Ah, for crying out loud, I sat on this for too long and we already have #19108 open... 😓 

I guess this is still valuable for the font size part though.

File: packages/block-library/src/navigation/edit.js
Comment (noahtallen) [inline packages/block-library/src/navigation/edit.js]: It feels wrong to have to do the inline styles all over the place. Is this a normal practice for font sizes in blocks?

**Review by noahtallen — APPROVED** [review]
The code looks good to me. It also tests great locally!

One thought -- I notice the paragraph block uses a new inline mechanism for font. Should we add this too?

Screen Shot 2019-12-16 at 4 54 00 PM

**Comment by Copons** [comment]
> The code looks good to me. It also tests great locally!
> 
> One thought -- I notice the paragraph block uses a new inline mechanism for font. Should we add this too?
> 
> Screen Shot 2019-12-16 at 4 54 00 PM

@noahtallen Oh, I missed that!
I'm putting this in WIP so that I can check it out and see how to integrate it here.

File: packages/block-library/src/navigation/edit.js
Comment (Copons) [reply packages/block-library/src/navigation/edit.js]: Inline styles are used for custom values that don't use predefined classes, so there's really no way around them.
They are also used for custom colors, etc.
Whenever there's a custom style attribute, we just have to use an inline style. 🙂

**Comment by noahtallen** [comment]
oh my 😁 my bad! That's some good stuff. We should have that

**Comment by noahtallen** [comment]
Who should we ping on this? I think it should be ready to go. I can't think of any reason to not have it.

**Comment by Copons** [comment]
@noahtallen I'm not sure, but the codeowners have been automagically pinged already. 🙂

**Review by talldan — APPROVED** [review]
This is working nicely, thanks for the addition.

It'd be great to remove that one classname before merging.

File: packages/block-library/src/navigation/edit.js
Comment (Copons) [reply packages/block-library/src/navigation/edit.js]: Oh good catch! Removed in 6789b26.

I'll merge as soon as tests are clear!

**Pull request merged**
